source/_integrations/switchbot.markdown

```diff
@@ -128,6 +130,15 @@ The blind tilt is exposed as a cover entity with control of the tilt position on
 
 The close button will close the blinds to the closest closed position (either 0% or 100%), and defaults to closing down if the blinds are fully open. Because Home Assistant believes 100% is open, the default cards will disable the open button when the tilt is at 100%, but the action will still work and open the blind to 50%.
 
+## SwitchBot Roller Shade
+
+The Roller Shade is exposed as a cover entity with control of the position only:
+
+| Tilt Position | Roller Shade State |
```

Comment:
```suggestion
| Tilt position | Roller Shade state |
```
We use sentence-style capitalization, also in headings. I leaft Roller Shade, as it looks like that's the product name.